Languages/Java/questions.json

```diff
@@ -151,6 +151,33 @@
       "trueOrFalse": false,
       "shortAns": "It is a problem that occurs while running the program",
       "tag": "Java"
+    },
+    {
+      "id": 17,
+      "question": "What is a key difference between Error and Exception?",
+      "details": "An Exception is recoverable while an Error is irrecoverable, example of error: IOError, OutOfMemoryError, example of exception: IllegalStateException, RuntimeException",
+      "questionType": "multi",
+      "trueOrFalse": false,
+      "shortAns": "An Exception is recoverable while an Error is irrecoverable",
```

Comment:
Please phrase the beginning of your short answer with "It is."

Do not include key words from the question in the answer.
Thank you for your participation @rishu1601